src/Desktop/UIAutomation/CustomObjects/Config.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+using System;
+using System.IO;
+
+namespace Axe.Windows.Desktop.UIAutomation.CustomObjects
+{
+    class Config
+    {
+        [JsonProperty("properties")]
+        public CustomProperty[] Properties { get; set; }
+
+        public static Config FromText(string text)
```

Comment:
Methods should begin with a verb. In this case "Create" would be appropriate because an object is being created and returned.

Reply:
This pattern also appears in other places, like [A11yElement](https://github.com/microsoft/axe-windows/blob/main/src/Core/Bases/A11yElement.cs#L685). It might be worth revising them all if we want to have a consistent pattern

Reply:
I agree, but this breaks precedent from names in other classes. Do we care?

Reply:
I personally prefer that we use best practices in newly added code over maintaining consistency. Others may have different points of view.